packages/sitecore-jss-react/src/components/BYOCRenderer.tsx

```diff
@@ -0,0 +1,90 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Just a minor thing
Do we need to exclude "any"? In general, seems like it's not used here and we need to avoid that according to our rules

Reply:
This is an omission from earlier work, thanks for pointing it out.